airbyte-workers/src/main/java/io/airbyte/workers/temporal/TemporalTraceConstants.java

```diff
@@ -0,0 +1,50 @@
+/*
+ * Copyright (c) 2022 Airbyte, Inc., all rights reserved.
+ */
+
+package io.airbyte.workers.temporal;
+
+/**
+ * Collection of constants for APM tracing of Temporal activities and workflows.
+ */
+public final class TemporalTraceConstants {
```

Comment:
@colesnodgrass I don't feel strongly about any of this, so happy to re-organize.  I am basically trying to avoid string duplication.  That being said, they are all being used for tracing/APM purposes.  I would tend to lean towards reversing the name order than having a separate class.  Thoughts?

Reply:
@colesnodgrass I went against my statement above and created a `Tags` class within the constants class.